contracts/ZKPVerifier.sol

```diff
@@ -51,40 +54,60 @@ contract ZKPVerifier is IZKPVerifier, Ownable {
         return true;
     }
 
-    function getZKPRequest(uint64 requestId)
-        external
-        view
-        override
-        returns (ICircuitValidator.CircuitQuery memory)
-    {
+    function getZKPRequest(
+        uint64 requestId
+    ) external view override returns (ICircuitValidator.CircuitQuery memory) {
         return requestQueries[requestId];
     }
 
     function setZKPRequest(
```

Comment:
This method `setZKPRequest` could easily reuse the functionality of the `setZKPRequestRaw` method to avoid DRY violation.

Reply:
@AndriianChestnykh I guess we're agreed to keep interfaces as it is for now...
@OBrezhniev @demonsh  @vmidyllic please correct me if I'm wrong